src/webfaf/filters.py

```diff
@@ -56,7 +57,7 @@ def fancydate(value, base_date=None):
 
     if old_date.month == base_date.month and old_date.year == base_date.year:
         # computes a number of calendar weeks (not only 7 days)
-        offset = round((d.days - base_date.isoweekday()) / 7, 0) + 1
+        offset = round((d.days - base_date.isoweekday()) // 7, 0) + 1
```

Comment:
This needs to be / so the round() works

Reply:
I think that the first operator is ```int``` or ```long``` and the second one is ```int```, so this commit is right to ensure the same behavior in the both Python 2 and 3 versions.

Reply:
Yes, you're right.